hello_world_attest_fn_call/README.md

```diff
@@ -132,7 +132,22 @@ So then to run our function, you can call:
 make run
 ```
 
-__Step 4: Extract function output from the Blocky AS attestation__
+The call to `bky-as attest-fn-call` will find and use
+[`./config.toml`](./config.toml) configuration file.
+Its `host` field, set to `local-server`, will direct `bky-as` to start a local, 
+non-TEE instance of the Blocky AS server to run our function in demo mode.
+You can confirm this by finding the following output:
+
+```
+🚀 Starting local server at http://126.0.0.1:8081 ...success
```

Comment:
Is this an error, shouldn't it be `127.0.0.1`?

Reply:
Yes, I have no idea how that pasted wrong!